docs/working-with-projects/working-with-images/preheat-images.md

```diff
@@ -0,0 +1,121 @@
+   The evaluation process will traverse the existing event-based preheat policies under the project where the target image bound in 
+   the event is pushed and check if the target image can match the pre-defined filters and criteria of some event-based 
+   preheat policies. The matched event-based preheat policies with fixed source image will be executed to complete the 
+   preheating process.
+   {{< /note >}}
+    
+1. Click **ADD** button to save the policy.
+
+## Manage Preheat Policy
+
+1. Go to **Projects** and open your project from the project list.
+1. Open the **P2P Preheat** tab, all the existing preheat policies are listed in the grid view with front selection enabled.
+   ![preheat policy list](../../../img/p2p-preheat/policy-list.png)
+1. Select the policy by checking the checkbox at front of the row, click **ACTIONS** to open the drop down menu.
+1. Click **Execute** to start the execution of the selected policy immediately.
+1. Click **Disable**/**Enable** to disable/enable the selected policy.
+  
+   {{< note >}}
+   Disabled policy cannot be executed.
+   {{< /note >}}
+  
+1. Click **Edit** to open the edit dialog and do modifications to the selected policy.
+1. Click **Delete** to delete the selected policy.
+  
+   {{< note >}}
+   If the executions of the selected policy are still in progress, the deletion will be rejected.
+   {{< /note >}}
+
+## Manage Executions of Preheat Policy
+
+1. Select the policy by clicking the radio button at the front of the row. If the policy has been executed before, the 
+relevant executions will be listed in the execution data grid.
+   ![policy execution](../../../img/p2p-preheat/policy-execution.png)
+1. For each execution, you can find the following data:
+    - ID: identity of the execution with a hyperlink pointing to the detailed page
+    - Status: `Success`,`Error` and `Running`
+    - Trigger: the trigger way of the execution, it can be `Manual`,`Scheduled` and `Event-based`
+    - Start Time: the start time of the execution (rendered as local time format)
+    - Duration: the overall duration of the execution
+    - Success Rate: each execution may contain multiple tasks, the percent of the success ones over the total
+    
+   {{< note >}}
+   For the `Error` status, there will be a small info icon with tooltip that containing the error message next to it.
+   For the `Success` status, if there are no images matching the filters and criteria defined in the policy, a small 
+   info icon with tooltip which indicates that no images to preheat will be placed next to it.
+   {{< /note >}}
+    
+1. Click the ID hyperlink to open the detailed page of the execution.
+   ![execution details](../../../img/p2p-preheat/execution-details.png)
+  
+   {{< note >}}
+   Each execution may preheat more than one matched images ,and each image will be preheated by one task separately. 
```

Comment:
fix spacing: "images ,and" "images, and"

Reply:
Do you mean preheating tasks in Harbor here? An execution record may container multiple preheating tasks because multiple images may meet the policy's criteria?

Reply:
yup